### PR TITLE
kind: switch to common.mk

### PR DESCRIPTION
**Pull request by jaxesn**
*Issue #, if available:*

*Description of changes:*
/hold

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by abhay-krishna** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/aws/eks-anywhere-build-tooling/pull/163#" title="Author self-approved">jaxesn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=aws%2Feks-anywhere-build-tooling).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/aws/eks-anywhere-build-tooling/blob/main/OWNERS)~~ [jaxesn]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: build/lib/eksd_releases.sh
Comment (abhay-krishna) [inline build/lib/eksd_releases.sh:39]: not a big deal, but should we remove the extra spaces here? I think there were two or three besides this one in the script.

File: projects/kubernetes-sigs/kind/patches/0001-Switch-to-AL2-base-image-for-node-image.patch
Comment (abhay-krishna) [inline projects/kubernetes-sigs/kind/patches/0001-Switch-to-AL2-base-image-for-node-image.patch:7]: Was this intentional?

File: projects/kubernetes-sigs/kind/build/node-image-build-args.sh
Comment (abhay-krishna) [inline projects/kubernetes-sigs/kind/build/node-image-build-args.sh:49]: Could we change these links to the `v0.11.1` tag, since it looks like some of these files/image tags in main have been bumped up since.

**Comment by abhay-krishna** [comment]
Looks great! Just some minor nits

**Comment by jaxesn** [comment]
nits fixed

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**
